packages/ember-data/lib/system/model/attributes.js

```diff
@@ -24,14 +24,16 @@ Model.reopenClass({
     Example
 
     ```javascript
+    import Ember from 'ember';
+    import DS from 'ember-data';
 
-    App.Person = DS.Model.extend({
+    var Person = DS.Model.extend({
```

Comment:
@sly7-7 the problem is the `Ember.get()` call below. I'm not sure how this is supposed to work with ember-cli now. Is there something similar that could replace it?